src/twisted/conch/test/test_endpoints.py

```diff
@@ -9,9 +9,12 @@
 from struct import pack
 from errno import ENOSYS
 
+import hamcrest
```

Comment:
This is bound to be controversial.

Unlike [testtools](http://testtools.readthedocs.io/en/latest/), [PyHamcrest](https://pyhamcrest.readthedocs.io) doesn't depend on pbr.  It style also conflicts with Twisteds.

However, it made the test a lot shorter and clearer.